test/test_linalg.py

```diff
@@ -54,6 +54,20 @@ def blaslt_supported_device():
             return True
     return False
 
+def set_tunableop_defaults():
+    if torch.cuda.is_available():
```

Comment:
Consider inverting the condition and than you don't need to keep the rest of the setup under the if condition
```suggestion
    if not torch.cuda.is_available():
        # TunableOp not supported on CPU at this time.
        return      
```